packages/headless/src/api/knowledge/post-answer-evaluation.ts

```diff
@@ -20,7 +20,7 @@ export interface AnswerEvaluationPOSTParams {
 
 export const answerEvaluation = answerSlice.injectEndpoints({
   endpoints: (builder) => ({
-    post: builder.mutation<void, AnswerEvaluationPOSTParams>({
```

Comment:
From the [RTK docs](https://redux-toolkit.js.org/rtk-query/usage/queries):

> A query operation can be performed with any data fetching library of your choice, but the general recommendation is that you only use queries for requests that retrieve data. For anything that alters data on the server or will possibly invalidate the cache, you should use a [Mutation](https://redux-toolkit.js.org/rtk-query/usage/mutations).

Calling the POST /evaluations endpoint seems to correspond to the 'anything that alters data on the server' condition, so it would appear that using mutation was indeed the correct approach here...

Reply:
Changing from a mutation to a query might have some side effects we do not fully control/understand.

This issue warrants a deeper investigation with an MRE and asking questions on RTK's GitHub about whether we think the behaviour is not kosher or do not fully understand it.

However, I do appreciate that this lengthy process could be unduly burdensome for a small fix like this.

So, I'm OK with the idea of shipping this, but I think we should have an ulterior investigation logged & planned.

Reply:
Here @fbeaudoincoveo greatly suspects that the call returning nothing while having its `content-type` header to `application/json` might be the cause of the query mutation throwing,

Unfortunatly, after being unsuccessful at trying to change the response `content-type` header of the create feedback post call (scalatra limitations), the plan is to merge this work around for now. 

We'll also leave a trace in the SVCAPI to leave a hint about potentially removing the content-type from the call.